sandy-city-utah/targetsolutions-user-sync
# feat(ad-users): pull ad users from api

### What Changed?
  - create basic application that pulls ad users from the api
  - copy some functionality from the API itself (configuration)
  - utilize the extracted functionality from types/csv export)
  - comment out the stuff out of scope for this specific ticket

### Issues Addressed:
  - P21312-15

### Issue Testing:
  - Manual Developer Testing

### Test Results:
1. Run the application
1. See that the console has the users retreived from the api (between 300-500, depending; might need to add pulling some of them from the database)
